### DISCUSSION TITLE
Link as rendered in the page for teaching-demos is incorrect

### DISCUSSION THREAD

**Discussion by mnardell** [carpentries/instructor-training#1522] [General]
I am not sure what is the underlying problem with how instructor-training page was written in MD, however the link intended to point to https://pad.carpentries.org/teaching-demos is instead being rendered as: https://carpentries.github.io/instructor-training/%7B%7Bpage.demopad%7D%7D

The markdown content where the link is found is one line 253 in the section " Sign-up and Set-up"
"To sign up, select a session that works for you on [the Instructor Training Demonstration Sessions Etherpad]({{page.demopad}}), and add"
.. I am not sure how the parameterized links work in MD, so I am not certain of how to fix.

**Comment by karenword** [comment]
thanks @mnardell ! This has now been fixed! #1531
